routers/repo/issue.go

```diff
@@ -515,7 +519,36 @@ func getFileContentFromDefaultBranch(ctx *context.Context, filename string) (str
 	return string(bytes), true
 }
 
-func setTemplateIfExists(ctx *context.Context, ctxDataKey string, possibleFiles []string) {
+func setTemplateIfExists(ctx *context.Context, ctxDataKey string, possibleDirs []string, possibleFiles []string) {
+	if ctx.Query("template") != "" {
```

Comment:
Is it only ctx.Query("template") templates that get to have the labels set? What about the files in L552-L559?

If those can have/should have labels/meta set then you could build a list of possible templates and then append the defaults to the end before iterating over them in a single loop.

Reply:
I'm not sure what to do in that case for compatibility.  
If it falls through to 552-559, we are dealing with a "legacy" issue template.

Should those be parsed for frontmatter, or treated as plain-text (markdown)?

If parsing for frontmatter, I suppose I shouldn't drop files with invalid frontmatter.

Reply:
Yeah it's probably simplest and least surprising for users if all the templates act the same way - so if there's a frontmatter block doesn't parse then just dump it as raw content and allow the legacy templates to have the additional yaml parsing. 

I appreciate that's technically breaking but you're adding functionality in lots of places then.

Reply:
Since it no longer drops invalid frontmatter, this technically isn't a breaking change as it will spit out whatever contents directly.